<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>أصناف الهدايا</string>
  </property>
  <widget class="QTableWidget" name="Qtable_redeem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>761</width>
     <height>271</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <column>
    <property name="text">
     <string>الباركود</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>الشركه</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>الفرع</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>النقاط المستحقه</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>تاريخ البدايه</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>تاريخ النهايه</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>الحاله</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(198, 207, 199);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>181</width>
       <height>92</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QComboBox" name="CMB_company"/>
      </item>
      <item>
       <widget class="QComboBox" name="CMB_branch"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>239</x>
       <y>10</y>
       <width>51</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>الشركة</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>الفرع</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>الباركود</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Qline_barcode">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>فترة البرنامج</string>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDateEdit" name="Qdate_from">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="displayFormat">
         <string>dd-MM-yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2020</year>
          <month>1</month>
          <day>3</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>من تاريخ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>60</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>الى تاريخ</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QDateEdit" name="Qdate_to">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2021</year>
          <month>12</month>
          <day>31</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>dd-MM-yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2021</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="Qbtn_search">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width:6em;
                                       }
                                        QPushButton:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                        QPushButton:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    
                                    </string>
        </property>
        <property name="text">
         <string>بحث</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Qbtn_search_all">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width:6em;
                                       }
                                        QPushButton:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                        QPushButton:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    
                                    </string>
        </property>
        <property name="text">
         <string>عرض الكل</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Qbtn_create">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width:6em;
                                       }
                                        QPushButton:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                        QPushButton:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    
                                    </string>
        </property>
        <property name="text">
         <string>إنشاء </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Qbtn_modify">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width:6em;
                                       }
                                        QPushButton:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                        QPushButton:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    
                                    </string>
        </property>
        <property name="text">
         <string>تعديل </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Qbtn_upload">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width:6em;
                                       }
                                        QPushButton:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                        QPushButton:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    
                                    </string>
        </property>
        <property name="text">
         <string>تحميل البيانات</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Qbtn_exit">
        <property name="styleSheet">
         <string notr="true">QPushButton{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width:6em;
                                       }
                                        QPushButton:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                        QPushButton:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    
                                    </string>
        </property>
        <property name="text">
         <string>خروج</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>الحالة</string>
    </property>
    <widget class="QRadioButton" name="Qradio_active">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>فعال</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Qradio_inactive">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>موقوف</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>النقاط المستحقه</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Qline_points">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
